tools/subgraph-example/package.json

```diff
@@ -31,7 +31,7 @@
     "ts-node": "^10.9.1",
     "typechain": "^5.2.0",
     "typescript": "^4.9.3",
-    "@hashgraph/hedera-local": "^2.3.2",
+    "@hashgraph/hedera-local": "^2.11.3",
```

Comment:
Here, if you try with `2.10.0` you might not need to specify the GAS.
maybe?

Reply:
@AlfredoG87 are you suggesting bumping down to work around a potential gas related regression so that this PR doesn't require the additional changes?
Presumably once the issue is resolved we can bump up to some > 2.11.3 version

Reply:
Won't work, because using lower version won't support synthetic contract results. Regression is in the relay, I think.

Reply:
Ohhh right, yeah... so we need to keep the latest version and use the workaround until the regression is resolved. 👍